### DISCUSSION TITLE
FlareSolverr communication with Intellipopup.com identified as SPAM

### DISCUSSION THREAD

**Discussion by K3v1n-M** [FlareSolverr/FlareSolverr#1030] [General]
I have FlarSolverr installed on unRAID.  My TPLink router identified it communicating with www(dot)intellipopup(dot)com and blocked the traffic because it identified it as SPAM.  Not sure if this is planned, but changing TEST_URL flag stopped the communication.

**Comment by ilike2burnthing** [comment]
Default is `https://www.google.com`, anything other than that is external to FlareSolverr.
